Files/UserControls/NavigationToolbar.xaml.cs

```diff
@@ -28,902 +28,120 @@
-                }
-            }
-        }
 
-        bool INavigationToolbar.IsEditModeEnabled
+        private bool isMultiPaneActive;
+
+        public bool IsMultiPaneActive
         {
             get
             {
-                return ManualEntryBoxLoaded;
+                return isMultiPaneActive;
             }
             set
             {
-                if (value)
-                {
-                    EditModeEnabled?.Invoke(this, new EventArgs());
-                    VisiblePath.Focus(FocusState.Programmatic);
-                    DependencyObjectHelpers.FindChild<TextBox>(VisiblePath)?.SelectAll();
-                }
-                else
+                if (value != isMultiPaneActive)
                 {
-                    ManualEntryBoxLoaded = false;
-                    ClickablePathLoaded = true;
+                    isMultiPaneActive = value;
+                    NotifyPropertyChanged(nameof(IsMultiPaneActive));
                 }
             }
         }
 
-        public ObservableCollection<ListedItem> NavigationBarSuggestions = new ObservableCollection<ListedItem>();
-
         private void VisiblePath_Loaded(object sender, RoutedEventArgs e)
         {
             // AutoSuggestBox won't receive focus unless it's fully loaded
             VisiblePath.Focus(FocusState.Programmatic);
             DependencyObjectHelpers.FindChild<TextBox>(VisiblePath)?.SelectAll();
         }
 
-        public bool CanRefresh
-        {
-            get
-            {
-                return Refresh.IsEnabled;
-            }
-            set
-            {
-                Refresh.IsEnabled = value;
-            }
-        }
-
-        public bool CanNavigateToParent
-        {
-            get
-            {
-                return Up.IsEnabled;
-            }
-            set
-            {
-                Up.IsEnabled = value;
-            }
-        }
-
-        public bool CanGoBack
-        {
-            get
-            {
-                return Back.IsEnabled;
-            }
-            set
-            {
-                Back.IsEnabled = value;
-            }
-        }
-
-        public bool CanGoForward
-        {
-            get
-            {
-                return Forward.IsEnabled;
-            }
-            set
-            {
-                Forward.IsEnabled = value;
-            }
-        }
-
-        string INavigationToolbar.PathControlDisplayText
-        {
-            get
-            {
-                return PathText;
-            }
-            set
-            {
-                PathText = value;
-                NotifyPropertyChanged(nameof(PathText));
-            }
-        }
 
         private void NotifyPropertyChanged([CallerMemberName] string propertyName = "")
         {
-            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+            //PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
         }
```

Comment:
We can probably completely remove this since it won't be implemented inside the control's code behind itself.